docusaurus/docs/React/release-guides/new-reactions.mdx

```diff
@@ -0,0 +1,176 @@
+# Introducing new reactions (with breaking changes)
+
+As a "move-away-from-EmojiMart" initiative we decided to start with reaction components (selector and list) and revamp the DX to make customization of these components much easier and more intuitive without having to rebuild default selector and list components from scratch.
+
+### Main reasons for a revamp
+
+- inability to reuse default (Stream supplied reactions) with your custom ones
+- strong reliance on EmojiMart and inability to use completely custom reactions out of the box
+- certain `additionalEmojiProps` did not work with Stream-supplied reactions
+
+```tsx
+// not exported directly (hidden, poor DX)
+import { defaultMinimalEmojis } from 'stream-chat-react/dist/components/Channel/emojiData';
+
+// this wouldn't work
+export const customReactions = [
+  {
+    // relies on EmojiMart-supplied sprite sheet, "native" option does not work for Stream-supplied reactions
+    // you'd need to look for supported id's in stream-emoji.json under "emojis" key
+    id: 'bulb',
+  },
+  // unsupported
+  {
+    id: 'rick_roll',
+  },
+  ...defaultMinimalEmojis,
+];
+```
+
+## New default setup and how it works
+
+SDK by default comes with six reaction types (`angry`, `haha`, `like`, `love`, `sad` and `wow`) which are newly rendered by [`StreamEmoji`]() component which utilises sprite sheet system and renders images for each reaction to make sure it works flawlessly on every system. Default reaction options are defined in [`defaultReactionOptions`]() and these options are reused for both [`ReactionSelector`]() and [`ReactionsList`]() (as well as [`SimpleReactionsList`]()). These options come by default from the ComponentContext but local component property will be prioritised if defined:
+
+```ts
+contextReactionOptions = defaultReactionOptions;
+// ...
+const reactionOptions = propsReactionOptions ?? contextReactionOptions;
+```
+
+## Custom reaction types and components
```

Comment:
I'd also include an example of how to override built-in reaction types with custom emoji